pandas/tests/series/methods/test_nlargest.py

```diff
@@ -127,8 +127,8 @@ def test_nsmallest_nlargest(self, s_main_dtypes_split):
     def test_nlargest_misc(self):
 
         ser = Series([3.0, np.nan, 1, 2, 5])
-        tm.assert_series_equal(ser.nlargest(), ser.iloc[[4, 0, 3, 2]])
-        tm.assert_series_equal(ser.nsmallest(), ser.iloc[[2, 3, 0, 4]])
+        tm.assert_series_equal(ser.nlargest(), ser.iloc[[4, 0, 3, 2, 1]])
+        tm.assert_series_equal(ser.nsmallest(), ser.iloc[[2, 3, 0, 4, 1]])
```

Comment:
can you rewrite this test using `result=` and `expected=` while making changes in this test.